%! BibTeX Compiler = biber
%TC:ignore
\documentclass{article}

\usepackage{xcolor, colortbl}
\definecolor{BLUELINK}{HTML}{0645AD}
\definecolor{DARKBLUELINK}{HTML}{0B0080}
\definecolor{LIGHTGREY}{gray}{0.9}
\PassOptionsToPackage{hyphens}{url}
\usepackage[colorlinks=false]{hyperref}
% for linking between references, figures, TOC, etc in the pdf document
\hypersetup{colorlinks,
    linkcolor=DARKBLUELINK,
    anchorcolor=DARKBLUELINK,
    citecolor=DARKBLUELINK,
    filecolor=DARKBLUELINK,
    menucolor=DARKBLUELINK,
    urlcolor=BLUELINK
} % Color citation links in purple
\PassOptionsToPackage{unicode}{hyperref}
\PassOptionsToPackage{naturalnames}{hyperref}

\usepackage{biorxiv}
\usepackage[backend=biber,eprint=false,isbn=false,url=false,intitle=true,style=nature,date=year]{biblatex}
\addbibresource{biblio.bib}

\usepackage{url}
\usepackage{amssymb,amsfonts,amsmath,amsthm,mathtools}
\usepackage{lmodern}
\usepackage{xfrac, nicefrac}
\usepackage{bm}
\usepackage{listings, enumerate, enumitem}
\usepackage[export]{adjustbox}
\usepackage{graphicx}
%\usepackage{bbold}
\usepackage{pdfpages}
\pdfinclusioncopyfonts=1
\usepackage{lineno}
\usepackage{tabu}
\usepackage{hhline}
\usepackage{multicol,multirow,array}
\usepackage{etoolbox}
\usepackage{booktabs}
\usepackage{makecell}
\usepackage{orcidlink}
\usepackage{csvsimple-l3}

\addbibresource{biblio.bib}

\newcommand{\NS}[1]{\textcolor{red}{\textbf{\emph{[NS: #1]}}}}

\newcommand{\UniDimArray}[1]{\bm{#1}}
\newcommand{\der}{\text{d}}
\newcommand{\e}{\text{e}}
\newcommand{\Ne}{N_{\text{e}}}
\newcommand{\proba}{\mathbb{P}}
\newcommand{\pfix}{\proba_{\text{fix}}}
\newcommand{\dn}{d_N}
\newcommand{\ds}{d_S}
\newcommand{\dnds}{\dn / \ds}
\newcommand{\Sphy}{S_{0}}
\newcommand{\SphyDel}{\mathcal{D}_0}
\newcommand{\SphyNeu}{\mathcal{N}_0}
\newcommand{\SphyBen}{\mathcal{B}_0}
\newcommand{\Sphyclass}{x}
\newcommand{\SphyclassAlt}{y}
\newcommand{\given}{\mid}
\newcommand{\Spop}{S}
\newcommand{\SpopDel}{\mathcal{D}}
\newcommand{\SpopNeu}{\mathcal{N}}
\newcommand{\SpopBen}{\mathcal{B}}
\newcommand{\ProbaPopDel}{\proba [ \SpopDel]}
\newcommand{\ProbaPopNeu}{\proba [ \SpopNeu ]}
\newcommand{\ProbaPopBen}{\proba [ \SpopBen ]}
\newcommand{\AdvMean}{\beta_b}
\newcommand{\DelMean}{\beta_d}
\newcommand{\thetaSyn}{\theta_{\text{S}}}
\renewcommand{\baselinestretch}{1.5}
\renewcommand{\arraystretch}{0.6}
\linenumbers
\frenchspacing

\title{Evolution of recombination rates under different mating system}

\author{
    \large
    \textbf{Thomas {Brazier}$^{1*}$\orcidlink{0000-0001-5990-7545} and Sylvain {Glémin}$^{1,2}$\orcidlink{0000-0001-7260-4573}}\\
    \normalsize
    $^{1}$University of Rennes, CNRS, ECOBIO (Ecosystems, Biodiversity, Evolution), Rennes, France\\
    $^{2}$Department of Ecology and Genetics, Evolutionary Biology Center and Science for Life Laboratory, Uppsala University, Uppsala, Sweden \\
    \textbf{Corresponding author:} \texttt{\href{mailto:thomas.brazier@univ-rennes.fr}{thomas.brazier@univ-rennes.fr}} \\
}


\textbf{Keywords:} \\



%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% ARTICLE START
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\begin{document}

\maketitle

\begin{abstract}
\end{abstract}


\newpage


\section*{Introduction}

Recombination considered as one of the main advantages of sexual life cycles
Genetic shuffling and mating system; effect on genetic shuffling
Efficiency of selection; new genetic combinations, variance of reproductive success, reducing linkage disequilibrium and
negative interference between loci (HRI)
Evolution of recombination itselves; evolution of the number of COs and recombination rates
"As the number of CO per chromosome does not vary much (typically from 1 to 3, 4; Fernandes et al. 2018) the difference in average recombination rate between chromosomes within a species is mainly explained by their size (Stapley et al., 2017; Haenel et al., 2018; Brazier and Gl ́emin, 2022)"
"Genetic recombination lies at the heart of the sexual life cycle, and is often considered as one of the main evolutionary benefits of sexual reproduction (Otto 2021)."

Pros and cons of recombination.

How does recombination rates relate to life history traits.

'Recombination rates can be influenced by environmental and demographic factors, but are also heritable and underpinned by specific genetic loci [16–20] and can respond to selection [21,22].Therefore, they have the potential to vary in a manner dependent on the evolutionary or selective contexts [6]. ' Stapley 2017


CO assurance vs. CO interference/homeostasy
Despite strong mechanistic constraints (proximate determinants) there is space to evolve between one and four COs per chromosome and meiosis

Heritability of recombination rates
Evolvability


Selection on recombination
Direct and indirect selection
Importance of rec in selfing species, selective interference
long- vs. short-term effects of recombination
On comprend mieux les mecanismes et contraintes moleculaires sur la recombination, mais "the evolutionary forces acting on recombination in natural populations remain elusive"
1 a 4 COs, espace pour evoluer vers plus ou moins de recombinaison
Besides species can evolve towards higher rec rate by increasing the number of chromosomes
Indirect selective forces, effect of recombination on genetic variation
"as recombination then increases the variance in fitness among offspring and the efficiency of natural selection"


Selfing
Effect of autofecondation on the evolution of recombination
Selection for recombination often higher in selfing specites; recombination less efficient to produce effetive shuffling
in highly homozygous species
Mutation load; increase in deleterious mutations must be counter-acted
What we know from theory
"The exchange of genetic material between individuals via various forms of sex and recombination has been hypothesised as a mechanism that has evolved at least in part to regenerate variance in fitness (Otto, 2009)."
West et al. 1999; Gouyon 1999; Otto 2009;
"costs associated with the mating process (Lehtonen et al., 2012)"
"indirect selection stems from the effect of recombination in breaking or creating genetic combinations."
"Although mechanical contraints may act on the number of COs, substantial variation in recombination rates can be observed between individuals of the same population. Indeed, variation in the genome-wide and/or local recombination rate between individuals has been described either in model species (e.g fruit flies; Singh et al. 2015), domesticated species (e.g. house mice, pigs, cattle, sheep, maize, honey bees; Dumont et al. 2009; Bauer et al. 2013; Ma et al. 2015; Petit et al. 2017; Kawakami et al. 2019; Brekke et al. 2022) or in natural populations (e.g. humans, red deer, Soay sheep, wild mice; Wang et al. 2017; Johnston et al. 2016, 2018; Halldorsson et al. 2019). This variation in recombination has a heritable genetic basis, with heritability ranging from 8% (Johnsson et al., 2021) to 48% (Dumont et al., 2009). Some of the genes found in the regions that contribute to variation in the recombination rate are shared among several mammals 1
such as PRDM9, RNF212 or HEI10. Therefore, heritable variation in recombination rates exists within populations, and natural selection could potentially act upon this variation."
"Moreover, a series of experiments have shown that recombination rates can evolve rapidly in response to strong selection applied on recombination rate itself or on other traits (reviewed in Otto and Barton 2001)."
In plants, change of mating system are recurrent, response to it? good system to study the effects of mating system
"The increase in recombination in response to strong selection could be interpreted as indirect selection on recombination, that is, as a way of increasing variation in fitness (Otto and Barton, 2001). It has long been hypothesised that sex and recombination are advantageous because they increase genetic variation and thus facilitate adaptation (Weismann, 1889). However, a large amount of theoretical work developed since the 60s has shown that this explanation is in fact not trivial: in particular, recombination does not always increase variation, and increasing variation is not always advantageous (reviewed by Otto 2009)."




What we know from previous empirical studies
Trends?
Observed positive correlations recombination/selfing?
GwRR. Chromosome size effect, genome size + relative size effect
Broad vs fine scale, effects of heterozygosity
"In addition, the relationship between recombination and selfing rate would benefit from being updated with new data from genetic maps in plants (but also other organisms) as the correlation has mainly been obtained from cytological data on the number of chiasmata per bivalent, that are only indirect measures of recombination rates."

Proximate causes of recombination rate variation such as chromosome size or gene density have been clearly identified (Brazier Glemin), yet 'key evolutionary (ultimate) causes and consequences of this variation' have received less attention.
proximate causes both at a broad (chromosome size, gene/repeat density) and fine scale (promoters, chromatin state, methylation, PRDM9 motifs)

'in plants Cf/B was higher in selfers compared to outcrosses [79,80].' Stapley 2017
79. Gibbs PE, Milne C, Carrillo MV. 1975 Correlation between the breeding system and recombination index in five species of Senecio. New Phytol. 75, 619–626. (doi:10.1111/j.1469-8137.1975.tb01428.x) 80. Zarchi Y, Simchen G, Hillel J, Schaap T. 1972 Chiasmata and the breeding system in wild populations of diploid wheats. Chromosoma 38,



"An interesting pattern observed in several genera of flowering plants is that self-fertilizing species tend to have higher chiasma frequencies than their outcrossing relatives (Roze and Lenormand 2005; Ross-Ibarra 2007). Detailed comparisons between the genetic maps of the selfing Arabidopsis thaliana and its outcrossing relative Arabidopsis lyrata also point to higher recombination rates in A. thaliana (Kuittinen et al. 2004; Hansson et al. 2006; Kawabe et al. 2006). A possible explanation for higher recombination rates in selfers could be that polymorphism between homologs hinders recombination (Borts and Haber 1987). However, this hypothesis does not stand up to closer scrutiny, as available data suggest that substantial levels of divergence are needed to prevent meiotic recombination (Chen and JinksRobertson 1999), while data from tetraploid rye (Benavente and Sybenga 2004) and from crosses between A. thaliana strains with different levels of divergence show that crossovers may occur preferentially in heterozygous regions (Ziolkowski et al. 2015; Blackwell et al. 2020), possibly due to a positive effect of mismatches among homologs on crossover initiation (Blackwell et al. 2020). Because recombination between homozygous loci has no genetic effect, selfing reduces the efficiency of recombination in breaking LD (Nordborg 2000; Wright et al. 2008), and one may thus expect that increased rates of recombination could evolve to compensate for this effect. However, indirect selection for recombination should vanish under complete selfing (as heterozygosity should then be extremely rare), and the effect of selfing on selection for recombination may thus be nonmonotonic. Furthermore, it is not immediately obvious that models for the evolution of recombination under random mating can be directly transposed to the case of partial selfers. Indeed, simulation models have shown that recombination may be favored under different conditions in partially selfing than in outcrossing populations, though the exact mechanisms remained unclear (Charlesworth et al. 1977, 1979; Holsinger and Feldman 1983)."

'Genome-wide recombination rates (cM/Mb) can vary more than tenfold across eukaryotes' yet most of this variation is driven by chromosome size. This strong association could hide more subtle associations, including evolution of recombination rates. Once controlled for chromosome size one can measure the residual variation in CO number among species and properly test evolutionary hypotheses for selection fo recombination.


Our approach
Meta-analytic approach
Comparative phylogenetic approach
Leverage variation in rec rate at a broad scale from pedigree-based maps
Therefore, they have the potential to vary in a manner dependent on the evolutionary or selective contexts [6].
Stapley 2017 was the first study to leverage high density genetic maps across hundreds of species in order to address how recombination itself evolved. Identified... effects
Previous empirical studies were restricted to chiasma count or comparisons between a few closely related species



Flowering plants are a good study system, variation in recombination rates, genome size and life history traits such as mating system and longevity.A wealth of high-quality genetic maps in plants.Broad taxonomic scale.



\section*{Materials and methods}

\subsection*{Recombination dataset}

The dataset contains a list of 207 species with their genome size, number of chromosomes and genetic map length plus breeding system and other life history traits.
The main aim is to test the hypothesis that selfing should select for higher recombination rates (Charlesworth et al. 1977,1979, Roze and Lenormand 2005, Stetsenko and Roze 2022).

sample size. Number of species

Stapley data has been curated and augmented
Controlled for SNP density and number of progeny
'In cases where a species had multiple maps, we chose the map with the most markers or the most individuals in cases where two maps had a similar number of markers.' Stapley 2017
But there is a SNP density effect
Average linkage map length instead?


Kew garden chromosome size




\subsection*{Life History Traits}


\subsection*{Phylogenetic regression}


\section*{Results}


Table avec nb d'especes, linkage map length, chromosome number, genome size, rec rate summary statistics pour chaque categorie + total


\section*{Discussion}


'To estimate the GwRR from linkage map data, we divided the linkage map length (the sum of the length of all sexaveraged LGs) by the haploid genome size (in Mb) (box 2 and figure 2). This is a commonly reported measure of recombination rate [11,23–27] and provides a useful metric to compare across taxa with vastly different genome sizes.'Stapley 2017, discuss this measure, we propose a slightly different way to measure rec rates, taking into account differences in chromosome szie and number across genomes
Indeed Stapley also note that 'This measure averages recombination across both the open and transcriptionally active euchromatic region and the closed and inactive heterochromatic regions of the genome. Recombination is often suppressed in heterochromatic regions, and the strength of suppression and the proportion of the genome that is heterochromatic vary greatly between organisms (see [69]). Thus, the GwRR represents a genome average that reveals differences in recombination rate, but will be related to differences in the amount of heterochromatin in the genome and how strongly recombination is suppressed in these regions. Taking account of the proportion of the genome that is heterochromatic may provide more informative estimates of recombination with respect to evolutionary processes [27,69];'
Our measure is directly comparable to chiasma count per bivalent
'Whether karyotypic variation is driven by selection on recombination rate is unclear (e.g. [64,65]), but Burt [66] demonstrated that an increase in the efficacy of selection was better achieved by increasing the number of COs per chromosome rather than increasing the number of chromosomes.' Stapley 2017
\end{document}

\begin{document}




\section*{Acknowledgments}

We wish to thank Laurent Duret, Pierre-Alexandre Gagnaire, Marie Raynaud, Julien Joseph, Nicolas Lartillot and all the other members of the HotRec ANR project, as well as members of the lab and people met outside for the discussion of the results. We used the GENOUEST computing facility for analyses. \textbf{Funding:} Agence Nationale de la Recherche, Grant ANR-19-CE12472 0019 / HotRec. \textbf{Author contributions:} Original idea: S.G.; Data analyses: T.B.; First draft: T.B.; Editing and revisions: T.B., S.G. \textbf{Competing interests:} The authors declare no conflicts of interest. \textbf{Data and materials availability:} Analysis scripts and documentation are available at \textbf{https://github.com/ThomasBrazier/landrec-gradients}. Data produced in this study are available at \textbf{https://osf.io/3aekw/}.


\printbibliography

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% SUPPLEMENTARIES
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%


\section*{Supplementary figures}

\renewcommand{\thefigure}{S\arabic{figure}}

\setcounter{figure}{0}


\begin{figure}[!htb]
  \includegraphics[width=0.9\textwidth]{figures/FigS1.jpeg}
  \centering
  \caption{Chromosome recombination maps. (A) Fine-scale recombination maps estimated with LDhat. Mean $\rho$/kb for each inter-SNP interval. (B) Broad-scale recombination maps (cM/Mb, 100 kb windows) estimated from Marey maps in \cite{brazierDiversityDeterminantsRecombination2022b}. The grey ribbon is the 95\% confidence interval estimated by 1,000 bootstraps on SNP markers.
  }
  \label{figure:FigS1}
\end{figure}



\clearpage

\section*{Supplementary tables}

\renewcommand{\thetable}{S\arabic{table}}

\setcounter{table}{0}

\begin{table}[h!]
\centering
\caption{Metadata of the twelve datasets. Dataset name used during analyses, author of the original study, year of the original study, mating system, number of SNP before filtering, number of SNPs after filtering, total number of individuals in the dataset, number of individuals sampled for LDhat analyses, accession of the reference genome, number of chromosomes, number of genes, public database where to access original data (if relevant), direct link to the original data (if relevant), doi of the original study.}
\label{table:TableS1}
\end{table}


\end{document}
